### PR TITLE
Implemented partially imported drop

### PR DESCRIPTION
**Pull request by jeeyi**
Sometimes some languages are delayed and the drop is partially imported. Current behavior is that the drop is considered to have `Imported` status as long as there is at least one xliff that was imported successfully. It's hard to distinguish between the fully completed drop and the partially completed drop. 

Partially completed drop should appear under `In Progress` filter with `Partially Imported` status. Also the detail should show which language was imported.

Screen Shot 2019-04-16 at 10 52 10 AM

### CODE REVIEW

**Comment by jeeyi** [comment]
Do not merge yet, one more change is coming - a button to force the `Partially Imported` status to `Imported` status.

File: webapp/src/main/java/com/box/l10n/mojito/entity/Drop.java [hunk 1]
```diff
@@ -96,6 +96,10 @@
     @JsonView(View.DropSummary.class)
     protected Boolean importFailed;
 
+    @JsonView(View.DropSummary.class)
+    @Column(name = "partiallyImported")
```
Comment (aurambaj) [inline webapp/src/main/java/com/box/l10n/mojito/entity/Drop.java]: `partially_imported`?

File: webapp/src/main/resources/db/migration/V44__Add_imported.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+alter table `drop` add column partiallyImported bit not null default false;
```
Comment (aurambaj) [inline webapp/src/main/resources/db/migration/V44__Add_imported.sql]: same as in the entity `partially_imported`

File: webapp/src/main/java/com/box/l10n/mojito/entity/Drop.java (hunk 1 above)
Comment (jeeyi) [reply webapp/src/main/java/com/box/l10n/mojito/entity/Drop.java]: Done

File: webapp/src/main/resources/db/migration/V44__Add_imported.sql (hunk 2 above)
Comment (jeeyi) [reply webapp/src/main/resources/db/migration/V44__Add_imported.sql]: Done

**Comment by jeeyi** [comment]
This is the button to mark partially imported drop to completed. This only shows up for the partially imported drops.

Screen Shot 2019-04-16 at 4 04 01 PM

File: webapp/src/main/resources/public/js/components/drops/Drops.js [hunk 3]
```diff
@@ -173,6 +173,15 @@ let Drops = React.createClass({
         this.delayGetNewRequests(500);
     },
 
+    /**
+     * Handle complete onclick event
+     */
+    onClickComplete(dropId) {
+        DropActions.completeRequest(dropId);
+
```
Comment (aurambaj) [inline webapp/src/main/resources/public/js/components/drops/Drops.js]: extra line?

Comment (jeeyi) [reply webapp/src/main/resources/public/js/components/drops/Drops.js]: Removed :)

**Pull request merged**